src/Mafin.Web.Api.Rest/Mafin.Web.Api.Rest.csproj

```diff
@@ -1,18 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net7.0;net6.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
```

Comment:
Can be `netstandard2.0` only.

Reply:
I thought about it a bit and I think you're right, it doesn't add any value as my implementation of PatchAsync is essentially a back-port anyways.

Updated to target `netstandard2.0` only, thanks.